src/main/features/response/routes/statement-of-means/dependants/education.ts

```diff
@@ -43,7 +42,6 @@ function renderView (form: Form<Education>, res: express.Response): void {
 export default express.Router()
   .get(
     page.uri,
-    FeatureToggleGuard.featureEnabledGuard('statementOfMeans'),
```

Comment:
why have you removed all these guards? shouldn't they now be using the admissions toggle instead?
you may also need to pass the claim.features object